1365.py

```diff
@@ -1,17 +1,11 @@
+import bisect
+
 class Solution:
     def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
         sorted_nums = sorted(nums)
         ans = []
         
         for num in nums:
-            idx = sorted_nums.index(num)
-            
-            while idx > 0:
-                if num > sorted_nums[idx - 1]:
-                    break
-                else:
-                    idx -= 1
-
-            ans.append(idx)
+            ans.append(bisect.bisect_left(sorted_nums, num))
```

Comment:
LGTM.

Just for the record, the following is what I would write if I didn't know `bisect.bisect_left()`:

```python 
def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
    lookup = sorted(nums)
    res = []
    
    for num in nums:
        res.append(lookup.index(num))    
    
    return res
```

It is slower, but the code is compact and correct.